meta/tastaturberuf/contao-image-copyright-bundle/de.yml

```diff
@@ -3,7 +3,7 @@ de:
     description: >
         Mit diesem Contao-Bundle kannst Du auf einfache Weise einen Lizenzhinweis zu allen Bildern hinzufügen und diese dann in einem Modul anzeigen lassen.
 
-        Besonders hilfreich wenn Du Stockfotos von diversen Anbietern verwendest und die Lizenzhinweisen anzeigen musst.
+        Besonders hilfreich wenn Du Stockfotos von diversen Anbietern verwendest und die Lizenzhinweise anzeigen musst.
```

Comment:
```suggestion
        Besonders hilfreich, wenn du Stockfotos von diversen Anbietern verwendest und die Lizenzhinweise anzeigen musst.
```

Reply:
Das große Du war beabsichtigt, is aber letztlich auch egal.